backend/benefit/applications/jobs/daily/daily_ application_jobs.py

```diff
@@ -13,4 +13,5 @@ class Job(DailyJob):
     help = "Django daily jobs are executed here."
 
     def execute(self):
```

Comment:
By the way, this file is called `daily_ application_jobs.py`, with a whitespace between the snakes 🧐 😂